src/Generators/FactoryGenerator.php

```diff
@@ -23,29 +21,11 @@
  *
  * @author Graham Campbell <graham@alt-three.com>
  * @author Scott Robertson <scottymeuk@gmail.com>
+ * @author Michael Bodnarchuk <davert@codeception.com>
  */
-class FactoryGenerator implements GeneratorInterface
+class FactoryGenerator extends EntityGenerator
 {
-    /**
-     * The kind of attribute that will be generated.
-     *
-     * @var string
-     */
-    private $kind;
-
-    /**
-     * The model instance.
-     *
-     * @var object
-     */
-    private $model;
-
-    /**
-     * The factory muffin instance.
-     *
-     * @var \League\FactoryMuffin\FactoryMuffin
-     */
-    private $factoryMuffin;
+    const PREFIX = 'factory|';
```

Comment:
> :-1: for dealing with prefixes like this

so you think that hardcoding 8 was better? https://github.com/thephpleague/factory-muffin/blob/master/src%2FGenerators%2FFactoryGenerator.php#L89

I don't think that prefix should be bound to class name.  At least CallableGenerator does not have any pefixes, so we can't stick to its name.